### ISSUE TITLE
Implement outputing Floats inside fold for the SQL backend

### ISSUE THREAD

**Issue by jcd2020** [kensho-technologies/graphql-compiler#655]
Currently not implemented for SQL

**Comment by pmantica1** [comment]
> Currently not implemented for SQL

What makes Floats special?

**Comment by obi1kenobi** [comment]
Does this raise `NotImplementedError` or error out some other way? Is this specific to a particular flavor of SQL, or does it not work for any of them?

**Comment by jecdohmann** [comment]
Error occurs on line 796 in to_sql for FoldedContextField. The fix should be as simple as just adding another branch to the if-elif statement

```
    def to_sql(self, aliases, current_alias):
        """Return a sqlalchemy Column picked from the appropriate alias."""
        # get the type of the folded field
        inner_type = strip_non_null_from_type(self.field_type.of_type)
        if GraphQLInt.is_same_type(inner_type):
            sql_array_type = 'INT'
        elif GraphQLString.is_same_type(inner_type):
            sql_array_type = 'VARCHAR'
        else:
            raise NotImplementedError('Type {} not implemented for outputs inside a fold.'.format(
                inner_type <--- ERROR HERE
            ))

        # coalesce to an empty array of the corresponding type
        empty_array = 'ARRAY[]::{}[]'.format(sql_array_type)
        return sqlalchemy.func.coalesce(
            aliases[
                self.fold_scope_location.base_location.query_path,
                self.fold_scope_location.fold_path
            ].c['fold_output_' + self.fold_scope_location.field],
            sqlalchemy.literal_column(empty_array)
        )
```

**Comment by chewselene** [comment]
Implemented in #796
